### PR TITLE
More traitor items Part 1

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Adds a number of previously random kits as well as new ones into a new uplink category at an increased price. Adds a new special bundle to the bundles category, The vent-crawling fiend, Added Holocarp for peoples who prefer their holoparasite to be ...more fishy.

## Why It's Good For The Game
Traitors are a bit weak, hoping to give a bit of a push with those things. 

## Changelog
:cl:
add: Added a new uplink category, Loadout along with many bundles for said category
add: Added holocarp for the peoples who prefer their holoparasite to be a spectral carp and the space assassin bundle!
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/moretraitoritems/code/modules/uplink/loadout_skyrat.dm [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/hacker
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/sniper
+	name = "Hitman bundle"
+	desc = "Carry out efficient and likely not-so-discreet assassinations with this unique sniping kit. Carrying case not included."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/sniper
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/metaops
```
Comment (MLGTASTICa) [inline modular_skyrat/modules/moretraitoritems/code/modules/uplink/loadout_skyrat.dm]: Horrible idea oh god

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/granters.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	ADD_TRAIT(user, (traits), type) //wtf
+	user.log_message("Gained the ability [traitname] ([traits])", LOG_ATTACK, color="orange")
+	onlearned(user)
+
+/obj/item/book/granter/traitsr/onlearned(mob/user)
+	..()
+	if(oneuse)
+		user.visible_message("<span class='warning'>You rip out the pages of the [src]!</span>")
+
+/obj/item/book/granter/traitsr/amogusbook
```
Comment (tf-4) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/granters.dm]: memey
```suggestion
/obj/item/book/granter/traitsr/ventcrawl_book
```

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/granters.dm [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	traits = TRAIT_VENTCRAWLER_ALWAYS
+	name = "Military Contortionist Guide"
+	traitname = "ventcrawling expert"
+	desc = "A special operations handbook for teaching people with at least a basic understanding of infiltration tactics how to most effectively utilize small spaces such as air ducts or pipes."
+	greet = "<span class='boldannounce'>You’ve mastered the art of climbing through air pipes!</span>"
+	icon_state = "stealthmanual"
+	remarks = list("Have a healthy diet..", "Know when to use vents to your advantage..", "Avoid being called suspicious...", "Best paired with close quarters skills...", "Pressure resistant gear recommended...")
```
Comment (tf-4) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/granters.dm:35]:
```suggestion
	remarks = list("Have a healthy diet..", "Know when to use vents to your advantage..", "Best paired with close quarters skills...", "Pressure resistant gear recommended...")
```

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 4]
```diff
@@ -0,0 +1,181 @@
+
+/obj/item/storage/box/syndie_kit/spaceassassin/PopulateContents()
+	new /obj/item/kitchen/knife/combat(src)
+	new	/obj/item/book/granter/traitsr/amogusbook(src)
```
Comment (tf-4) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]:
```suggestion
	new	/obj/item/book/granter/traitsr/ventcrawl_book(src)
```

**Comment by CameronWoof** [comment]
Remove the ventcrawl book. Unrestricted ventcrawl is going to be the most busted and irritating thing imaginable. I do not want to open that can of worms.

File: modular_skyrat/modules/moretraitoritems/code/modules/uplink/loadout_skyrat.dm [hunk 5]
```diff
@@ -0,0 +1,99 @@
+/datum/uplink_item/bundles_tc/spaceassassin
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/moretraitoritems/code/modules/uplink/loadout_skyrat.dm]: Can you move these unlink items to the uplink_items.dm in master files?

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm [hunk 6]
```diff
@@ -95,3 +112,88 @@
...
+	name = "Hitman bundle"
+	desc = "Carry out efficient and likely not-so-discreet assassinations with this unique sniping kit. Carrying case not included."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/sniper
+	cost = 25
+
+/datum/uplink_item/loadout_skyrat/metaops
+	name = "Bulldog Operative bundle"
+	desc = "Fight the power with this frontline combatant kit, featuring armor and armaments commonly utilized by assault operative teams."
+	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/metaops
+	cost = 25
```
Comment (KathrinBailey) [inline modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]:
```suggestion
	cost = 28
```
**This is the minimum acceptable TC cost, assuming it's even acceptable to begin with.**
Bulldog = 15
Syndie Hardsuit = 8 (23 total)
Emag = 4 (27 total)
Airlock Emag = 3 (30 total)
C4 = 1 (31 total)
C4 = 1 (32 total)
Microbomb Implant = 2 (34 total)
Microbomb Implant if we were to price it for traitors = 5-ish (37-ish)

We also wouldn't give the microbomb to traitors for merely 2TC. I'm a little concerned on offering a "haahaa gib myself" option to traitors too, but I'll put that concern aside for now.

Comment (dawsonkeyes) [reply modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]: This kit is actually a direct copy-and-paste of the Meta-Ops random kit you can currently get ingame for 20tc. While I understand limiting the overall costs of kits here as they aren't random, why haven't we thus changed how the random kits work?

**Comment by Gandalf2k15** [comment]
Check over the suggestions.

**Comment by Gandalf2k15** [comment]
We can test merge this.

**Comment by Azarak** [comment]
Remove the vent crawl book

**Comment by Gandalf2k15** [comment]
Seems good.

**Pull request merged**